package.json

```diff
@@ -47,7 +52,10 @@
     "selenium-webdriver": "3.3.0",
     "sinon": "^4.0.1",
     "travis-multirunner": "^4.0.0",
+    "ts-node": "^4.0.1",
+    "tslint": "^5.8.0",
     "twilio": "^2.11.1",
+    "typescript": "^2.6.2",
```

Comment:
I think you should pin to a specific version of TypeScript, since, from what I understand, they do not follow SemVer.

Reply:
Also, I'm not sure if this has implications for ts-node and/or tslint.